### PR TITLE
Update cue_notes_create.lua

### PR DESCRIPTION
**Pull request by cv-on-hub**
Added stem freezing "away_from_middle" and extra menu item "Cue Notes Flip Frozen". (additional coding by @asherber)

### CODE REVIEW

File: src/cue_notes_create.lua [hunk 1]
```diff
@@ -54,9 +75,10 @@ function show_error(error_code)
         empty_region = "Please select a region\nwith some notes in it!",
         no_notes_in_source_layer = "The music selected contains\nno notes in layer " .. config.source_layer,
         first_make_expression_category = "You must first create a new Text Expression Category called \""..config.cue_category_name.."\" containing at least one entry",
+        no_cue_notes = "The region selected contains \nno cue notes in layer " .. config.cuenote_layer
     }
     local msg = errors[error_code] or "Unknown error condition"
-    finenv.UI():AlertNeutral("script: " .. plugindef(), msg)
+    finenv.UI():AlertInfo("script: " .. plugindef(), msg)
```
Comment (asherber) [inline src/cue_notes_create.lua:81]: @cv-on-hub Shouldn't the `msg` param come first?

Comment (cv-on-hub) [reply src/cue_notes_create.lua:81]: The **second** string parameter (the optional "Title") is bigger and bolder. I wan't the "message" to be bigger and bolder so place it second. In this usage, at least, I'm only adding the name of the script as a courtesy. Not so important.

Comment (asherber) [reply src/cue_notes_create.lua:81]: It's up to you, of course, but on Windows the dialog title is neither bigger nor bolder. It also doesn't support line breaks, so you get text like "regionwith". And from a usability perspective, it seems unusual to look at the title bar of a dialog to get the important information.

Comment (cv-on-hub) [reply src/cue_notes_create.lua:81]: Thanks for letting me know. That's really ugly. Will remove the script name and move the message to the place where Windows can handle it.

Comment (asherber) [reply src/cue_notes_create.lua]: Thanks, lgtm

**Review by rpatters1 — APPROVED** [review]
lgtm

**Pull request merged**